docs/troubleshooting.md

```diff
@@ -62,3 +64,63 @@ For instance, try this:
 # oc create --raw /apis/admission.hive.openshift.io/v1/dnszones -f config/samples/hiveadmission-review-failure.json -v 8 | jq
 ```
 
+## Regaining access to a cluster after changing the API certificate
+
+There are several ways to regain access to a cluster after its API certificate has been changed.  Usually this will manifest as an error message similar to: `x509: certificate signed by unknown authority`.
+
+### Replacing the API certificate of a managed single cluster with kubeconfig
+
+In order for a new certificate for a cluster to be accepted by the system, it must be added to the kubeconfig, alongside or replacing any existing certificates. Start by inspecting your current kubeconfig:
+
+```sh
+$ export MY_KUBECONFIG_NAME=$(oc get cd $CLUSTERNAME -o "jsonpath={.spec.clusterMetadata.adminKubeconfigSecretRef.name}")
+$ oc get secret $MY_KUBECONFIG_NAME -o "jsonpath={.data.kubeconfig}" | base64 --decode > my-kubeconfig-file
```

Comment:
FYI there's an `oc extract` command that looks slightly less magical. Exercise for the reader to figure out how it works :)